templates/monorepo/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "mono",
+  "private": true,
+  "devDependencies": {
+    "@types/react": "^16.9.43",
+    "@types/react-dom": "^16.9.8",
+    "lerna": "^3.15.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "tsdx": "^0.13.2",
+    "typescript": "^3.9.7"
+  },
+  "workspaces": [
+    "packages/*"
+  ],
+  "scripts": {
+    "lerna": "lerna",
+    "dev": "lerna run start --stream --parallel",
+    "test": "lerna run test --",
+    "build": "lerna run build",
+    "prepublish": "lerna run prepublish",
+    "git-reset": "git reset --hard HEAD",
+    "git-clean": "git clean -d -x -e node_modules -e packages -f",
+    "now-build": "npm run build-website"
```

Comment:
What's this?

Reply:
These are helpers that I forgot to remove from Formik. Will get rid of the website stuff, but the others I actually think of are likely worth keeping.